testsuite/api/Makefile

```diff
@@ -14,6 +14,8 @@ deleteReferencesInSSD1.lua \
 deleteReferencesInSSD2.lua \
 deleteResourcesInSSP1.lua \
 deleteResourcesInSSP2.lua \
+getDirectionalDerivative_CS.lua \
+getDirectionalDerivative_ME.lua \
```

Comment:
The test is not part of this pull request. Maybe you forgot to add the file?

Reply:
Yes you are correct `getDirectionalDerivative_ME` is missing , i will add it

Reply:
this is fixed with this commit https://github.com/OpenModelica/OMSimulator/pull/1109/commits/12630fe699e544e6f1c3c1969cd661c5ef79d0a2